### PR TITLE
Fixed some doc warnings

### PR DESCRIPTION
**Pull request by lionello**
Also see #4689 for reducing the number of false positives. There are still some left, will work on that.

Can we add ddoc warning checks to the auto-tester please? At least prevent it from increasing.

### CODE REVIEW

File: std/conv.d [hunk 1]
```diff
@@ -5527,15 +5527,15 @@ unittest
  *      Random access range with slicing and everything
  */
 
-auto toChars(ubyte radix = 10, Char = char, LetterCase letterCase = LetterCase.lower, T)(T value)
```
Comment (Hackerpilot) [inline std/conv.d]: Why change this template argument instead of the doc comment?

**Comment by Hackerpilot** [comment]
The rest of the changes look good to me.

File: std/conv.d (hunk 1 above)
Comment (lionello) [inline std/conv.d]: No good reason. I thought all template params where PascalCased, but seems values aren't.

File: std/string.d [hunk 2]
```diff
@@ -2184,15 +2184,15 @@ auto representation(Char)(Char[] s) @safe pure nothrow @nogc
  * See_Also:
  *      $(XREF uni, toCapitalized) for a lazy range version that doesn't allocate memory
  */
-S capitalize(S)(S s) @trusted pure
+S capitalize(S)(S input) @trusted pure
```
Comment (Hackerpilot) [inline std/string.d]: It's really annoying that this module uses `input`, `s`, and `str` seemingly at random, but this isn't a problem with your pull request.

**Comment by Hackerpilot** [comment]
Auto-merge toggled on

**Pull request merged**